test/requirements_aistore.txt

```diff
@@ -1,2 +1,2 @@
+aistore >= 1.0.2
```

Comment:
I would set this to specific version. Otherwise, if something changes in the AIStore's API the tests will break.